modules/ticket_selector/assets/ticket_selector.js

```diff
@@ -56,7 +56,14 @@ jQuery(document).ready(function ($) {
                         $ticket_selector_row.removeClass('ee-hidden-ticket-tr');
                         active_rows++;
                     } else {
-                        $ticket_selector_row.addClass('ee-hidden-ticket-tr').find('.ticket-selector-tbl-qty-slct').val(0);
+						$ticket_selector_row.addClass( 'ee-hidden-ticket-tr' );
+						$qty_input = $ticket_selector_row.find(
+							'.ticket-selector-tbl-qty-slct'
+						);
+						// set qty to zero for non-radio inputs
+						if ( $qty_input.attr( 'type' ) !== 'radio' ) {
```

Comment:
Is there the possibility of checkbox inputs?  If so, should this account for them as well? Should it be a whitelist check instead of a blacklist check?

Reply:
no the Ticket Selector only uses select inputs or radio buttons.

Reply:
@tn3rb We had a request for using checkboxes for the ticket selector and I threw a snippet together that swapped out the select for a checkbox an then used the first value of the select if the checkbox was selected.

So would it be worth swapping this out? No objects if not, just noting that checkboxes have been requested/used previously.